autoendpoint/src/middleware/sentry.rs

```diff
@@ -22,22 +23,46 @@ pub fn sentry_middleware(
         scope.add_event_processor(Box::new(move |event| process_event(event, &sentry_request)))
     });
 
+    let state = request
```

Comment:
nit: It's probably not hurting anything as is but we might wanna avoid cloning the entire `ServerState` unless it's necessary. You could just pull out and clone solely the `metrics` instead.

Reply:
I'll admit, I'm being a bit lazy here. `.cloned()` does a clone if `app_data()` returns Some. That saves a bit of complexity around dealing with `.app_data()` returning `None` for any reason.